modules/raycast/lightmap_raycaster.cpp

```diff
@@ -33,6 +33,7 @@
 // From Embree.
 #include <math/vec2.h>
 #include <math/vec3.h>
+#include <xmmintrin.h>
```

Comment:
Note that this won't compile on `!SSE2` AFAIK. That's currently guarded in `config.py` since we only support x86_64 currently, but eventually this will need to be improved (not necessarily in this PR).

Reply:
And it would probably crash if run on non-SSE capable CPU (e.g. 32 bit x86). I assume from above you don't build in the lightmapper into 32 bit builds, or something like this. Mind you you'd be hard pressed to find a non-SSE1 capable CPU these days lol.

Reply:
Yeah both Embree and OIDN only support x86_64 currently, so `modules/denoise` and `modules/raycast` are both only compiled on this arch. There's an aarch64 fork for Embree but we have yet to investigate using it (for macOS aarch64 mostly): https://github.com/lighttransport/embree-aarch64

Reply:
Yeah, I'll be taking a look at embree-aarch64 since we will need it for 4.0's occlusion culling too. But in the meantime this change should not cause any issues given it's disabled on anything other than x86_64.